Src/AutoFixture.xUnit.net/FreezeAttribute.cs

```diff
@@ -0,0 +1,145 @@
+using System;
+using System.Reflection;
+using Ploeh.AutoFixture.Kernel;
+
+namespace Ploeh.AutoFixture.Xunit
+{
+    /// <summary>
+    /// An attribute that can be applied to parameters in an <see cref="AutoDataAttribute"/>-driven
+    /// Theory to indicate that the parameter value should be frozen and used to satisfy any requests
+    /// made through the same <see cref="IFixture"/> and that match a set of criteria.
+    /// </summary>
+    [AttributeUsage(AttributeTargets.Parameter, AllowMultiple = false)]
+    public sealed class FreezeAttribute : CustomizeAttribute
```

Comment:
Agreed. I've merged `FreezeAttribute` into the `FrozenAttribute` (6dc0e02) and deprecated `As` (700b7f7).